pt.ipl.isel/courses.yaml

```diff
@@ -0,0 +1,74 @@
+school: pt.ipl.pt
```

Comment:
@xploitedd & @grimord check if this is aligned with what we've discussed.

Reply:
Yep, I think that's it. The only thing missing is the schools file which we can add once these are validated.

Reply:
Looks good!